grace/models/feature_extractor.py

```diff
@@ -173,14 +173,27 @@ def forward(
             bbox_image = F.pad(bbox_image, padding, "constant", 0)
 
             bbox_image = self.transforms(bbox_image)
-            bbox_image = self.normalize_func(bbox_image)
+            # bbox_image = self.normalize_func(bbox_image)
 
-            if self.training:
-                bbox_image = self.augmentations(bbox_image)
+            # if self.training:
+            #     bbox_image = self.augmentations(bbox_image)
 
-            bbox_image = self.normalize_func(bbox_image)
+            # bbox_image = self.normalize_func(bbox_image)
+
+            # Extract the features from ResNet-152 model:
             features = self.model(bbox_image)
             node_attrs[GraphAttrs.NODE_FEATURES] = features.squeeze()
 
+            # HACK:
+            # Average the 4x4 central pixel values of each node patch:
```

Comment:
same question as above, why keeping these comments?

Reply:
I'm not sure how to approach this - when testing different configs & hyperparameters, it may be useful to keep an option to just use the central patch average intensity, which is informative for the synthetic dataset. It's much faster compared to running each patch through an entire Resnet-152, which takes about 30-40s per single image. What do you reckon is the best to do here?